Tests/LibWeb/Text/input/Streams/ReadableByteStream-byob-commit-pull-into-descriptor.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<script src="../include.js"></script>
+<script>
+    asyncTest(async (done) => {
```

Comment:
Maybe it makes sense to `./Meta/import-wpt-test.py` the WPT test that was written of https://github.com/web-platform-tests/wpt/pull/48085 instead?